core/primitives/src/receipt.rs

```diff
@@ -131,9 +131,7 @@ impl BorshDeserialize for ReceiptEnum {
         match ordinal {
```

Comment:
I believe without the feature flags this fn is now equivalent to the one derived from BorshDeserialize trait (see comment on L129).